packages/bundle-source/src/zip-base64.js

```diff
@@ -108,6 +108,35 @@ export async function bundleZipBase64(
     };
   }
 
+  /**
+   * @param {import('@endo/compartment-mapper/src/types.js').Language} parser
+   * @param {Uint8Array} sourceBytes
+   * @param {string} specifier
+   * @param {string} location
+   * @param {string|import('source-map').RawSourceMap} sourceMap
+   */
+  const transformModuleSource = async (
+    parser,
+    sourceBytes,
+    specifier,
+    location,
+    sourceMap,
+  ) => {
+    if (!['mjs', 'cjs'].includes(parser)) {
+      throw Error(`Parser ${parser} not supported in evadeEvalCensor`);
+    }
+    const babelSourceType = parser === 'mjs' ? 'module' : 'script';
```

Comment:
It occurs to me that you could pass `'module'` and `'script'` without this translation.

Reply:
we need to indicate both parser lang and `babelSourceType`, so i decided to abstract over babel